f2/azure-pipelines.yaml

```diff
@@ -11,22 +12,24 @@ trigger:
       - api/*
       - f2/manifests/*
 
+# Use an Ubuntu VM to run Azure pipeline
 pool:
   vmImage: 'ubuntu-16.04'
 
+# Run NPM install to install depedencies on VM
 steps:
   - task: Npm@1
     inputs:
       command: 'install'
       workingDir: 'f2'
-
+# Test npm compile and other npm scripts in Azure pipeline
```

Comment:
The npm compile step is where we build the lingui files for internationalization. Poorly named now that I think of it. Maybe for the comment here just say something like “run automated tests”.

Reply:
Done